tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -2,8 +2,10 @@ const modalPage = require('./modal.wdio.page');
 const constants = require('@constants');
 const aboutPage = require('@page-objects/default/about/about.wdio.page');
 
-const hamburgerMenu = () => $('#header-dropdown-link');
-const userSettingsMenuOption = () => $('[test-id="user-settings-menu-option"]');
+// const hamburgerMenu = () => $('#header-dropdown-link');
+const hamburgerMenu = () => $('aria/Application menu');
+// Adding close side bar menu as hamburgerMenu will be hidden - comment to be removed
```

Comment:
I didn't fully understand. When will the comment be removed? 
The code should be self-explained as much as possible (clear function names, clear logic) so it doesn't need comments. 
Or, if this is a ToDo comment, then prefix with `ToDo` so the IDE will highlight it in the code and then link to the ticket where it will be fixed. For example:
```
// ToDo: Fix XYZ, Ref: https://github.com/medic/cht-core/[ticket number]
```